src/js/control-bar/control-bar.js

```diff
@@ -67,7 +68,8 @@ ControlBar.prototype.options_ = {
     'descriptionsButton',
     'subsCapsButton',
     'audioTrackButton',
-    'fullscreenToggle'
+    'fullscreenToggle',
+    'pictureInPictureToggle'
```

Comment:
I'd expect the fullscreen toggle to still be the right-most button, so, this would have to go above it.
I'm still a bit concerned on whether this can have issues, particularly in ad-plugins that often create their own control bar if it's included by default.

Reply:
Fullscreen toggle button is now the right-most button. I've updated screenshot as well.